test/remote/gateways/remote_flex_charge_test.rb

```diff
@@ -139,6 +158,16 @@ def test_successful_refund
     assert_equal 'DECLINED', refund.message
   end
 
+  def test_successful_void
+    @cit_options[:phone] = '998888'
+    set_credentials!
+    response = @gateway.authorize(@amount, @credit_card_mit, @cit_options)
+    assert_success response
+
+    assert void = @gateway.void(@amount, response.authorization)
+    assert_success void
```

Comment:
When I run this test on my local environment I get this error, should I take anything else into account to perform this test?

```
  "errors"=>
         [{"item1"=>"Failed to refund order",
           "item2"=>"REFUND_FAILED",
           "item3"=>"RefundFailed",
           "item4"=>true}],
```

Reply:
Ahhh yes void 'should' exactly like a refund but FlexCharge hasn't activated it yet